docs/explanations/architecture-2.md

```diff
@@ -0,0 +1,239 @@
+---
+sidebar_label: Architecture - 2
+sidebar_position: 61
+title: Kubewarden architecture
+description: The Kubewarden architecture
+keywords: [kubewarden, kubernetes, architecture]
+doc-persona: [kubewarden-all]
+doc-type: [explanation]
+doc-topic: [architecture]
+---
+
+<!--
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/explanations/architecture"/>
+</head>
+-->
+
+Kubewarden is a Kubernetes policy engine.
+It uses policies written in a programming language of your choosing.
+This language must generate a WebAssembly binary for Kubewarden to use.
+
+Some possible things for inclusion below.
+All top-level information for a Kubewarden developer, a policy developer, a manager?
+
+## Design principles
+
+- First design principle
+
+  Explanation
+
+- nth design principle
+
+  Explanation
+
+## What is a policy
+
+A policy is ???????. Sample as YAML?
```

Comment:
A policy is a OCI artifact that contains a webassembly module (the policy code) and the metadata required by the PolicyServer to perform admission request validations/mutations.